docs/examples/layout/Flex/README.md

```diff
@@ -1 +1 @@
-undefined
\ No newline at end of file
+Flex provides a `View`-like component that allows for quick creation of flex-based containers.
```

Comment:
`View`-like is pretty `native` specific descriptor -- maybe make clear that Flex is just a generic primitive/tag to be used across environments for flexing purposes?

Reply:
would it be better to say `provides a container that...`?

Reply:
Yep, that sounds good imo